plugins/setGT.c

```diff
@@ -113,6 +114,7 @@ const char *usage(void)
         "           X       .. allele with bigger read depth as determined from FMT/AD\n"
         "           p       .. phase genotype (0/1 becomes 0|1)\n"
         "           u       .. unphase genotype and sort by allele (1|0 becomes 0/1)\n"
+        "           i       .. inverse phase of genotype (0|1 becomes 1|0)\n"
```

Comment:
I am not a native speaker, but I think a better description would be "invert the genotype phase"